10/src/stylesheets/partials/content.sass

```diff
@@ -0,0 +1,45 @@
+@import "../variables/layout.sass"
+
+#content
+    margin-top: $header-height;
+
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
```

Comment:
You can use here the shortcut `flex-flow: row nowrap`. And these are values by default.